src/test/java/uk/gov/hmcts/sptribs/citizen/event/CicSubmitCaseEventTest.java

```diff
@@ -83,20 +84,26 @@ class CicSubmitCaseEventTest {
     @Mock
```

Comment:
public methods can be moved to the top in CicSubmitCaseEvent class

Reply:
Created https://tools.hmcts.net/jira/browse/DTSSTCI-366 to resolve this.